### PR TITLE
confirmation page: external link icons

### PR DESCRIPTION
**Pull request by cantsin**
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/555)

## What does this change?

- External link icons for confirmation page

### Author

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] Check for [accessibility](/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/cts_forms/templates/forms/confirmation.html [hunk 1]
```diff
@@ -135,10 +135,10 @@ <h3 class="intake-section-title">
             {% trans "Legal aid offices or members of lawyer associations in your state may be able to help you with your issue." %}"
             <ul class="confirmation-list">
               <li>
-                {% trans 'American Bar Association, visit <a href="http://www.findlegalhelp.org">www.findlegalhelp.org</a> or call <a href="tel:800-285-2221">(800) 285-2221</a>' %}
+                {% trans 'American Bar Association, visit <a class="external-link" aria-label="www.findlegalhelp.org" href="http://www.findlegalhelp.org/">www.findlegalhelp.org</a> or call <a href="tel:800-285-2221">(800) 285-2221</a>' %}
               </li>
               <li>
-                {% trans 'Legal Service Corporation (or Legal Aid Offices), visit <a href="https://www.lsc.gov/find-legal-aid">www.lsc.gov/find-legal-aid</a> or call <a href="tel:202-295-1500">(202) 295-1500</a>' %}
```
Comment (LindsayYoung) [inline crt_portal/cts_forms/templates/forms/confirmation.html:141]: lsc.gov/find-legal-aid redirects to https://www.lsc.gov/what-legal-aid/find-legal-aid are we sure we don't want to send people staight there?

Comment (cantsin) [reply crt_portal/cts_forms/templates/forms/confirmation.html:141]: Good question. Looks like there's a 301 redirect. Note that we might need to change this url in two places -- confirmation and landing. What do you want to do here?

**Comment by cantsin** [comment]
Doing some refactoring wrt external link icon SCSS.

**Comment by cantsin** [comment]
After some thought, I kept the original links as-is, thanks for the catch. Merging.

**Pull request merged**